package.json

```diff
@@ -197,9 +197,9 @@
           [
             "babel-plugin-styled-components",
             {
-              "displayName": true,
               "pure": true,
-              "namespace": "bufferapp/ui"
+              "namespace": "bufferapp/ui",
```

Comment:
Hey @gmzjuliana - won't the forward-slash break the namespace just as it happened with the SC_ATTR on #333? Since it's used as part of the class name, I thought it might not parse as a correct HTML attribute value.

Reply:
good one! doesn't seem to break it but I'll change it so it's inline with the rest. Good catch @msanroman ! 🌮